Ghidra/Processors/MIPS/data/languages/mips32Instructions.sinc

```diff
@@ -1644,7 +1644,13 @@ define pcodeop SYNC;
 	build RTsrc;
 	tmp:$(REGSIZE) = sext(simmed:2) + RTsrc;	
 	JXWritePC(tmp); 
-    call [pc];
+    goto [pc];
+}
+
+:jic RTsrc, simmed                                      is $(AMODE) & REL6=1 & prime=0x36 & jsub=0x00 & RTsrc & simmed & immed=0x00 & rt=0x1f {
+        build RTsrc;
```

Comment:
Could you fix the formatting on these two lines to make the indention match?

Reply:
Fixed, but the whole file is still mixed with tab and space.